REFAC_UI_PLAN.md

```
@@ -0,0 +1,100 @@
+            <Card.Body><BalancesTable /></Card.Body>
+         </Card>
+      </Card.Group>
+   </Section>
+</Page>
+```
+
+## Mapping Principles to Tactics
+| Principle | Tactic |
+|-----------|--------|
+| Use fewer borders | Prefer subtle shadow + contrast difference; borders only for separation in dense zones |
+| Establish hierarchy with size/weight/color | Use heading scale + accent color sparingly for primaries |
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 5; Actual: 13; Style: 1/2/3

(MD029, ol-prefix)